tests/specs/cli.ts

```diff
@@ -5,7 +5,7 @@ import { tsx, tsxPath } from '../utils/tsx';
 import { ptyShell, isWindows } from '../utils/pty-shell';
 
 export default testSuite(({ describe }, fixturePath: string) => {
-	describe('CLI', ({ describe, test }) => {
```

Comment:
Why's this removed? It's used on https://github.com/IlyaSemenov/tsx/blob/8b3ec3939876c3dd0077854f2fa9e74378f57654/tests/specs/cli.ts#L62

Reply:
My bad. I disabled that tests as it was failing on my machine, and didn't realise _that_ in turn was causing the linter error above.